semantic_release/commit_parser/angular.py

```diff
@@ -29,9 +29,11 @@ def _logged_parse_error(commit: Commit, error: str) -> ParseError:
 
 # types with long names in changelog
 LONG_TYPE_NAMES = {
-    "feat": "feature",
+    "build": "build system",
     "docs": "documentation",
```

Comment:
```suggestion
    "ci": "continuous integration",
    "docs": "documentation",
```

I guess `"CI"` would be updated to "Ci" in the changelog.